src/box/sql.h

```diff
@@ -376,6 +376,8 @@ struct vdbe_field_ref {
 	uint32_t data_sz;
 	/** Count of fields in tuple. */
 	uint32_t field_count;
+	/** Format that match data in field data. */
+	struct tuple_format *format;
```

Comment:
You might want to either nullify this field in `vdbe_field_ref_create()` or make it take the format argument always. Otherwise you have it not initialized and potentially with a garbage value in `vdbe_field_ref_prepare_data()` in `OP_Column`.

Reply:
Thanks, fixed.